src/content_script/index.tsx

```diff
@@ -167,9 +169,10 @@ async function showPopupThumb(text: string, x: number, y: number) {
 }
 
 document.addEventListener('mouseup', (event: MouseEvent) => {
-    window.setTimeout(() => {
-        const text = (window.getSelection()?.toString() ?? '').trim()
-        showPopupThumb(text, event.pageX + 7, event.pageY + 7)
+    window.setTimeout(async () => {
+        const text = (window.getSelection()?.toString() ?? '').trim();
```

Comment:
Why do not `await utils.getSettings()` in this function?

Reply:
Done. Please Take A Look.